CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## UNRELEASED
+
+### Adds
+
+* Accessibility improved for navigation inside modals and various UI elements.  
+Pages/Docs Manager and Doc Editor modal now have a better keyboard accessibility by
+keeping the focus on elements inside modals and giving it back to their parent modal when closed.
+
```

Comment:
feel free to rephrase it to a better English 😶

Reply:
"a better" => "better"

"... better keyboard accessibility. They keep the focus on elements inside modals and give it back..."